runtime.md

```diff
@@ -66,13 +65,15 @@ This operation MUST generate an error if it is not provided the ID of a containe
 This operation MUST return the state of a container as specified in the [State](#state) section.
 In particular, the state MUST be serialized as JSON.
 
+After a user launches a container using the [start](#start) operation, state requests by the same user in the same namespaces with the same runtime installation MUST successfully return that container's state by the time the [pre-start hooks](config.md#prestart) are called (step (3) in the [lifecycle](#lifecycle)) through the final [post-stop hooks](#config#poststop) (step (8) in the [lifecycle](#lifecycle)).
+Runtimes MAY return state under other conditions, for example to a different user or when asked from a different namespace.
```

Comment:
On Mon, Feb 22, 2016 at 05:41:51PM -0800, Doug Davis wrote:
> > +After a user launches a container using the [start](#start) operation, state requests by the same user in the same namespaces with the same runtime installation MUST successfully return that container's state by the time the [pre-start hooks](config.md#prestart) are called (step (3) in the [lifecycle](#lifecycle)) through the final [post-stop hooks](#config#poststop) (step (8) in the [lifecycle](#lifecycle)).
> > +Runtimes MAY return state under other conditions, for example to a different user or when asked from a different namespace.
> 
> This text should just be focused on scope of access to the ops - its
> not really about state. GeState() is just one op, this access stuff
> applies to all ops.

Where I have it (in the “Query State” section), it is just about state
accesss.  For other operations, access depends on state access (write
access for ‘start’ and read access for all other operations), as
described here [1].  There may be other access requirements for the
other operations, but those can be listed separately from these
state-retrieval requirements (and I don't think we currently list any
such requirements).

[1]: https://github.com/wking/opencontainer-specs/commit/2fff5454fb7f20beec6110ff8725fc8507f4cce8#diff-b84a8d65d8ed53f4794cd2db7e8ea731R53